var/spack/repos/builtin/packages/reframe/package.py

```diff
@@ -23,7 +23,7 @@ class Reframe(Package):
     maintainers = ['victorusu', 'vkarak']
 
     version('master',   branch='master')
-    version('3.0',       sha256='c4fe84a92d961546e4d0e33ca3476ba0d4cebc908eb4e33897f646bd1fd5205b')
+    version('3.0',       sha256='fb76b4513c03b84f5b3bbbc988f7747e5b58f04c983b3935bab1f2e81adccb82')
```

Comment:
Why this change in `sha256`?

Reply:
For some strange reason, also the current version of the package in the develop branch has the wrong hash and does not install.

Reply:
When making the recipe I used another version of ReFrame because the 3.0 was not released yet. I forgot to update the checksum to point to the correct one when ReFrame 3.0 was released.